app/tests/archives_tests/test_views.py

```diff
@@ -849,3 +849,25 @@ def test_archive_item_add_value(
             )
     assert response.status_code == 200
     assert ArchiveItem.objects.get().values.first().value
+
+
+@pytest.mark.django_db
+def test_api_archive_item_deletion(client):
```

Comment:
Needs tests for object based permissions, e.g.
- ensuring that a user can only delete an archive item if they have the "delete_archite_item" bit 
- ensuring that the only users and groups with the "delete_archite_item" bit are the editors group for this reader study
See usages of `get_groups_with_set_perms`